src/components/chip-set/chip-set.tsx

```diff
@@ -370,7 +394,7 @@ export class ChipSet {
                 {this.value.map(this.renderInputChip)}
                 <input
                     tabIndex={INPUT_FIELD_TABINDEX}
```

Comment:
Sorry for being off topic to the PR, but are we still setting tab-index to 1 here? tab-index should only ever be set to `-1` or `0`. I pointed this out before, but can't remember if there was some reason we couldn't just set it to `0`? @Kiarokh

Reply:
I don't know really. But I made this issue now so we don't forget to test this: https://github.com/Lundalogik/lime-elements/issues/1741